client/src/css/Login.css

```diff
@@ -0,0 +1,43 @@
+.login-wrapper {
```

Comment:
we can try to use [Hack4Impact Bridge](https://bridge-ui.now.sh) for easier component styling and functionality but for now this is good!